util/src/main/java/io/camunda/zeebe/util/sched/future/ActorFuture.java

```diff
@@ -54,4 +54,13 @@
   boolean isCompletedExceptionally();
 
   Throwable getException();
+
+  @Override
+  default void accept(final V value, final Throwable throwable) {
```

Comment:
:x: I think we also defined it on `CompletableActorFuture`, so we could now remove this and that `CompletableActorFuture` also implements `BiConsumer`.